exercises/E-conditionals/exercise.js

```diff
@@ -8,6 +8,13 @@
 
 let name = "Daniel";
 let danielsRole = "mentor";
+if (danielsRole === "mentor"){
+  console.log("Hi, I'm " + name + ", I'm a mentor.");
+} else {
+  console.log("Hi, I'm " + name + ", I'm a student.");
+}
+
+
```

Comment:
Nice work Oguzhan, 

This is just a simple suggestion, but instead of "I'am a Student." You could write:
"I'am a " + danielsRole + "."